cargo/private/runfiles_maker/runfiles_maker.rs

```diff
@@ -61,7 +61,7 @@ fn main() {
         .expect("Failed to create output directory");
         std::fs::copy(src, &out_dest).unwrap_or_else(|e| {
```

Comment:
If this is going to exist long term then perhaps this directory could have symlinks generated into it whenever possible and the files from `DefaultInfo.default_runfiles.files` can be tracked in addition to the output directory. This would probably match runfiles directories more directly but I'm concerned about how remote caches will handle a directory of symlinks so will save such an optimization for another time.